INSTALL.md

```diff
@@ -110,7 +110,7 @@ Several options can be passed to CMake, among which:
   values are `ON` and `OFF`),
   - `-DFAISS_ENABLE_PYTHON=OFF` in order to disable building python bindings
   (possible values are `ON` and `OFF`),
-  - `-DFAISS_ENABLE_RAFT=ON` in order to enable building the RAFT implementations
```

Comment:
What about lines 9, 20-21, 27, 38-39? Will we have a faiss-gpu-cuvs package?

Reply:
I intentionally did not make changes to the docs (readme and install) because I thought that is something entirely under FAISS' governance. Don't want to modify that without approval from you guys. But to answer your question: yes it will be called `faiss-gpu-cuvs`.

Reply:
Those lines reference packages distributed by the `nvidia` channel so if you have `faiss-gpu-cuvs` prepared we can coordinate the release.